.cirrus.yml

```diff
@@ -25,9 +25,9 @@ env:
     FEDORA_CONTAINER_FQIN: "registry.fedoraproject.org/fedora:32"
     PRIOR_FEDORA_CONTAINER_FQIN: "registry.fedoraproject.org/fedora:31"
 
-    # VM Images are maintained in the libpod repo.
-    _BUILT_IMAGE_SUFFIX: "podman-6530021898584064" 
-    FEDORA_CACHE_IMAGE_NAME: "fedora-32-${_BUILT_IMAGE_SUFFIX}"
+    # VM Image built in containers/automation_images
+    _BUILT_IMAGE_SUFFIX: "c6110627968057344"
+    FEDORA_CACHE_IMAGE_NAME: "fedora-${_BUILT_IMAGE_SUFFIX}"
```

Comment:
just triple checking, we don't need to hardcode the version number `-32-` any more?  If not, woot!

Reply:
Nope...'course the down-side is it's hard to answer "which version am I using".  Hopefully that's only a minor inconvenience.

Reply:
Nope == Don't need to hard-code version number